src/tfrnnlm/tensorflow-rnnlm.cc

```diff
@@ -0,0 +1,376 @@
+// tensorflow-rnnlm-lib.cc
+
+// Copyright 2017           Hainan Xu
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <utility>
+#include <fstream>
+
+#include "tensorflow/core/public/session.h"
+#include "tensorflow/core/platform/env.h"
+#include "tensorflow/core/protobuf/meta_graph.pb.h"
+
+#include "tfrnnlm/tensorflow-rnnlm.h"
+#include "util/stl-utils.h"
+#include "util/text-utils.h"
+
+namespace kaldi {
+using std::ifstream;
```

Comment:
Just tried cpplint and it seems
using some_scope::some_function is OK.

cpplint is actually complaining about thing like
"using namespace kaldi", citing "Do not use namespace using-directives.  Use using-declarations instead.", but I guess we're already using that in probably every kaldi binary, so maybe just keep it as it is.